src/components/Dashboard/TodoList.vue

```diff
@@ -227,6 +227,32 @@
                   >
                     <span>Resume</span>
                   </v-btn>
+                  <v-menu offset-y left>
```

Comment:
This code adds the menu for all correction types, but delete only applies to IA correction right?

Reply:
This would allow the deletion of any Correction Draft for Staff.

Reply:
Yes, but since other corrections are currently a manual process, I don't know if the BE handles this correctly.

Reply:
Can staff save Director corrections or Address Corrections as Drafts?

Reply:
Hmm, it looks like they can. Weird that there's only a Resume button and not a Delete option.
Check with Kaine/Linda/Scott on whether they want this.
Also, does it work with the existing BE?

Reply:
It does! I do remember Linda confirming they want to be able to delete Drafts, but i will double check 👍